### PR TITLE
Initial Codespace for Webaverse

### PR DESCRIPTION
**Pull request by AGoblinKing**
Allows easy running of the Webaverse App via Codespaces.

image

Whitescreen pending: https://github.com/webaverse/localstorage/pull/2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
\ No newline at end of file
+# Webaverse Codespace
+Run your own Webaverse on Codespaces
+
+## Quick Start
+
+ - [ ] Open a terminal and do the following:
+ 
+```sh
+# You will answer some cert questions
```
Comment (avaer) [inline README.md]: Can we skip the questions with flags?

Comment (avaer) [reply README.md]: Also can we automate the script entirely so the user does not need to run it? Like as part of the install?

**Review by avaer — COMMENTED** [review]
Looks good, just a couple of questions.

File: README.md (hunk 1 above)
Comment (AGoblinKing) [reply README.md]: yeah I had an init script initially in an earlier commit but I wasn't sure if we wanted to skip the questions on the certs. I'll make these changes.

**Comment by AGoblinKing** [comment]
Okay this will automatically initialize now and be ready for [F5]

**Review by avaer — APPROVED** [review]
LGTM

**Comment by AGoblinKing** [comment]
Now just to draw the rest of the owl for multiplayer.

**Pull request merged**